src/services/renown/index.ts

```diff
@@ -66,12 +66,16 @@ export class Renown {
                 credential,
             };
 
-            try {
-                const ens = await getEnsInfo(user.address, user.chainId);
-                user.ens = ens;
-            } catch (error) {
-                console.error(error);
-            }
+            getEnsInfo(user.address, user.chainId)
```

Comment:
We no longer wait to get the ens info (user name and image) before setting the user.
The ens request sometimes takes a long time and it was delaying the login process.